### PR TITLE
Add code of conduct, badges, improve README

### PR DESCRIPTION
**Pull request by doug-wade**
Address https://github.com/redfin/react-server/issues/29

### CODE REVIEW

**Comment by doug-wade** [comment]
[needz moar badges](https://github.com/boennemann/badges) -- maybe codeclimate? coveralls? powered by hipsters?

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -6,3 +6,6 @@ Agreement](https://cla-assistant.io/redfin/react-server). The purpose
 of this license is to protect contributors, Redfin, as well as users
 of this software project. Signing this agreement does not affect your
 rights to use your contributions for any other purpose.
+
+This project adheres to the [Open Code of Conduct][code-of-conduct]. By participating, you are expected to honor this code.
+[code-of-conduct]: http://todogroup.org/opencodeofconduct/#react-server/doug.wade@redfin.com
```
Comment (gigabo) [inline CONTRIBUTING.md]: What's the hash fragment at the end of this URL for?

File: README.md [hunk 2]
```diff
@@ -1,6 +1,18 @@
 # react-server monorepo [![Build Status][build-badge-img]][build-url]
 
-(see: [README](packages/react-server/README.md))
+# Monorepo?
+
+It's a thing, now.  All the cool kids are doing it, like [React](https://github.com/facebook/react/tree/master/packages), [Meteor](https://github.com/meteor/meteor/tree/devel/packages), [Ember](https://github.com/emberjs/ember.js/tree/master/packages), and [Babel](https://github.com/babel/babel/tree/master/packages).  All of the packages are versioned in lockstep, which makes it easier for us to make strong guarantees about the interoperability of the packages that work together in the react-server ecosystem.
+
+# react-server?
+The best place to start is [react-server's README](packages/react-server/README.md)), though you will likely want to read all of [react-server's docs](packages/react-server/docs).
```
Comment (gigabo) [inline README.md]: An extra `)` snuck in after the README.md URL.

File: README.md [hunk 3]
```diff
@@ -1,6 +1,18 @@
...
+
+# Contributing
+
+See [the contributing guide](CONTRIBUTING.md).
 
 [build-badge-img]: https://travis-ci.org/redfin/react-server.png?branch=master
 [build-url]: https://travis-ci.org/redfin/react-server
+[![npm version](https://badge.fury.io/js/react-server.svg)](https://badge.fury.io/js/react-server)
+[![Dependency Status](https://david-dm.org/redfin/react-server.svg)](https://david-dm.org/redfin/react-server)
+[![devDependency Status](https://david-dm.org/redfin/react-server/dev-status.svg)](https://david-dm.org/redfin/react-server#info=devDependencies)
```
Comment (gigabo) [inline README.md]: Seems like these badges make more sense at a package level.  Our monorepo `package.json` is private.  It doesn't correspond to any package on `npm`.

File: CONTRIBUTING.md (hunk 1 above)
Comment (doug-wade) [inline CONTRIBUTING.md]: I dunno, that's just what http://todogroup.org/opencodeofconduct/ spat out at me

File: README.md (hunk 3 above)
Comment (doug-wade) [inline README.md]: I'd say per [React's example](https://github.com/facebook/react), let's keep the version one in this README, but move the dependencies ones to their package's README. Seem reasonable?

File: CONTRIBUTING.md (hunk 1 above)
Comment (gigabo) [inline CONTRIBUTING.md]: Aha:

image

image

Comment (gigabo) [inline CONTRIBUTING.md]: Maybe that email address should be to a group?

File: README.md (hunk 3 above)
Comment (gigabo) [inline README.md]: Yes, though maybe useful to _also_ put version badges in the packages?

Not all packages are published with each version.

File: CONTRIBUTING.md (hunk 1 above)
Comment (doug-wade) [inline CONTRIBUTING.md]: Here's some examples:
opensource+codeofconduct@twitter.com (https://engineering.twitter.com/opensource/code-of-conduct)
oss-conduct@box.com (http://opensource.box.com/code-of-conduct/)
opensource@fb.com (https://code.facebook.com/pages/876921332402685)

so

opensource@redfin.com ?

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -6,3 +6,6 @@ Agreement](https://cla-assistant.io/redfin/react-server). The purpose
 of this license is to protect contributors, Redfin, as well as users
 of this software project. Signing this agreement does not affect your
 rights to use your contributions for any other purpose.
+
+This project adheres to the [Open Code of Conduct][code-of-conduct]. By participating, you are expected to honor this code.
+[code-of-conduct]: http://todogroup.org/opencodeofconduct/#react-server/opensource@redfin.com
```
Comment (gigabo) [inline CONTRIBUTING.md]: Thanks for getting this group set up.

**Comment by gigabo** [comment]
LGTM.  I guess we should get our dependencies up to date in packages before adding badges there. :)

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -6,3 +6,6 @@ Agreement](https://cla-assistant.io/redfin/react-server). The purpose
 of this license is to protect contributors, Redfin, as well as users
 of this software project. Signing this agreement does not affect your
 rights to use your contributions for any other purpose.
+
```
Comment (davidalber) [inline CONTRIBUTING.md]: I think we should give other people, who are not watching this repository, a chance to weigh in on the code of conduct before we merge.

**Comment by doug-wade** [comment]
@gigabo what's scope creep, anyway?

File: CONTRIBUTING.md (hunk 5 above)
Comment (doug-wade) [inline CONTRIBUTING.md]: @davidalber lemme know when you've completed whatever offline review you feel is necessary.

File: CONTRIBUTING.md (hunk 4 above)
Comment (davidalber) [inline CONTRIBUTING.md]: When I go to the code of conduct, the top of the page has this now:

image

File: CONTRIBUTING.md (hunk 5 above)
Comment (davidalber) [inline CONTRIBUTING.md]: Done. Ship it! The Contributor Covenant one (c127c1e), that is.

**Pull request merged**